portfolio/src/main/webapp/script.js

```diff
@@ -26,7 +26,13 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-
+git 
```

Comment:
git this out of here :)

This still happens to me when I forget to switch tabs and think I'm typing into a terminal but am still in my text editor haha.

Reply:
Oh wow I gotta watch out for that, I fixed it in the next commit.